contracts/transmuter/libraries/LibHelpers.sol

```diff
@@ -73,7 +73,8 @@ library LibHelpers {
     /// @dev The values in the `xArray` must be increasing
     function piecewiseLinear(uint64 x, uint64[] memory xArray, int64[] memory yArray) internal pure returns (int64) {
         uint256 indexLowerBound = findLowerBound(true, xArray, 1, x);
-        if (indexLowerBound == xArray.length - 1) return yArray[xArray.length - 1];
+        if (indexLowerBound == 0 && x < xArray[0]) return yArray[0];
```

Comment:
Is it still useful with the check in `checkFees`?

Reply:
I think we're better off leaving it in the global `piecewiseLinear` function -> because this one is meant as a library people can fork I guess